packages/uniforms-antd/src/DateField.tsx

```diff
@@ -2,7 +2,7 @@ import DatePicker from 'antd/lib/date-picker';
 import React, { Ref } from 'react';
 import moment from 'moment';
 import { connectField, filterDOMProps, Override } from 'uniforms';
-import { DatePickerProps } from 'antd/lib/date-picker/interface';
+import { DatePickerProps } from 'antd/es/date-picker';
```

Comment:
Are types exported from the `'antd'` root package? We could use that, as TypeScript removes type-only imports anyway.

Reply:
No, they are not available there. We can import `DatePickerProps` from `antd/lib/date-picker` just like `DatePicker` what I missed.